p2p/protocols/accounting.go

```diff
@@ -71,7 +71,7 @@ type Price struct {
 // A protocol provides the message price in absolute value
 // This method then returns the correct signed amount,
 // depending on who pays, which is identified by the `payer` argument:
-// `Send` will pass a `Sender` payer, `Receive` will pass the `Receiver` argument.
+// Sending will pass a `Sender` payer, receiving will pass the `Receiver` argument.
```

Comment:
I thought that by changing the text to use the actions (Sending/Receiving) instead of the actual former function names I had achieved just that. 

To me this is thus already fine and expresses what you are asking for, but if you insist I can change it.

Reply:
on second thought i think you're right, nevermind